### ISSUE TITLE
[Proj4] `proj4.fs`를 수정하고 이를 불러오는 방법

### ISSUE THREAD

**Issue by sunhongmin225** [swsnu/osspr2022#124]
안녕하세요. 이번 프로젝트에서는 `proj4.fs`라는 ext2 기반 file system을 담은 파일을 제출하도록 되어 있습니다. 그리고 요구사항으로 `proj4.fs`가 at least 2 directories with 2 files in each directory, and at least 2 files outside those directories를 포함하도록 하라고 되어 있습니다.

해당 요구사항을 만족하기 위해 `proj4.fs`를 어떻게 수정해야 하는지 헷갈려서 질문드립니다. 아래 명령어로 `proj4.fs`를 만들면 아직 아무것도 들어있지 않은 빈 file system이 담긴 파일이 만들어지는 것 같습니다.

```console
os@ubuntu:proj4$ dd if=/dev/zero of=proj4.fs bs=1M count=1
os@ubuntu:proj4$ sudo losetup /dev/loopXX proj4.fs
os@ubuntu:proj4$ sudo ../e2fsprogs/misc/mke2fs -I 256 -L os.proj4 /dev/loopXX
os@ubuntu:proj4$ sudo losetup -d /dev/loopXX
```

그 다음에 QEMU 내에서 `proj4.fs`를 mount시킨 `/root/proj4` 안에 필요한 files나 directories를 추가하는 식으로 `proj4.fs`를 수정하는 것이 맞나요? 이게 맞다면 해당 수정이 반영된 `proj4.fs`를 어떻게 불러올 수 있는지 잘 모르겠습니다. 잘못 이해하고 있는 부분이 있다면 지적해주시면 감사하겠습니다.

감사합니다.

**Comment by t-hyun** [comment]
네. 그렇게 수정하시면 됩니다! 그 뒤 QEMU를 끝내고 수정된 proj4.fs를 꺼내어 제출 내역에 포함시켜 주시면 됩니다.

**Comment by t-hyun** [comment]
QEMU 내부에서 umount를 하여야 하는데 제가 설명에서 잊은 것 같습니다 :) 혹시 그 부분을 빠뜨리신 것이 아닐까요?

`mount -o loop -t ext2 proj4.fs /root/proj4`
modification of /root/proj4
`umount proj4`
end QEMU
`mkdir mntdir`
`sudo mount tizen-image/rootfs.img mntdir`
`sudo cp mntdir/root/proj4.fs ./proj4.fs`

**Comment by sunhongmin225** [comment]
네 말씀해주신대로 umount를 하고 나니 생각한대로 잘 동작합니다. 도움주셔서 감사합니다!
